File: cmd/analyze/competitor-example/launchDarkly/java/sample-LD.kt
val visitor1 = FeatureFlag.newVisitor("visitor_1")
            .context(hashMapOf("age" to "32", "isVIP" to true))
            .hasConsented(true)
            .isAuthenticated(true)
            .build()

val visitor1 = FeatureFlag.newVisitor("visitor_1").build()
visitor1.updateContext("isVip", true)
visitor1.fetchFlags().invokeOnCompletion {

    val btnColorFlag = visitor1.stringVariation("LD-string-flag-kotlin", "#000000")
    val btnSizeFlag = visitor1.intVariation("LD-number-flag-kotlin", 12)
    val showBackgroundFlag = visitor1.boolVariation("LD-bool-flag-kotlin", false)

    val btnColorFlag = visitor1.stringVariationDetail("LD-string-flag-kotlin-1", "#000000")
    val btnSizeFlag = visitor1.intVariationDetail("LD-number-flag-kotlin-1", 12)
    val showBackgroundFlag = visitor1.boolVariationDetail("LD-bool-flag-kotlin-1", false)
}

File: cmd/analyze/code-samples/launchDarkly/net/sample-LD.cs
var visitor = Ff.NewVisitor("<VISITOR_ID>")
  .IsAuthenticated(true)
  .HasConsented(true)
  .WithContext(new Dictionary<string, object> {
    ["isVIP"] = true,
    ["country"] = "NL",
    ["loginProvider"] = "Google"
    })
  .Build();

  await visitor.FetchFlags();

var flag = visitor.StringVariation("LD-string-flag-cs", context, 'red');
var flag = visitor.IntVariation("LD-int-flag-cs", context, 13);
var flag = visitor.BoolVariation("LD-bool-flag-cs", context, true);

var flag = visitor.StringVariationDetail("LD-string-flag-cs", context, 'red');
var flag = visitor.IntVariationDetail("LD-int-flag-cs", context, 13);
var flag = visitor.BoolVariationDetail("LD-bool-flag-cs", context, true);